ape_solidity/_utils.py

```diff
@@ -23,6 +23,18 @@ class Extension(Enum):
     SOL = ".sol"
 
 
+class StandardErrors():
```

Comment:
I think this could probably subclass both `Enum` and `ape.exceptions.ContractError` to hook better into the `ape.reverts` manager, and extend whatever handling we have for compiler-specific errors

@antazoey will have a better idea what to do, but this is a good start! Thank you!